enterprise/backend/src/metabase_enterprise/sandbox/models/params/field_values.clj

```diff
@@ -75,6 +76,26 @@
                                          (mbql.u/match-one v [:dimension [:field field-id _]] field-id))]
                     {k (get login-attributes k)})))])))
 
+(defenterprise field-id->field-values-for-current-user
+  "Fetch *existing* FieldValues for a sequence of `field-ids` for the current User. Values are returned as a map of
```

Comment:
Can you clarify the expectation here? The docstring stresses this fetches `*existing*` FieldValues. It seems the unsandboxed fields behave this way (`params.field-values/default-field-id->field-values-for-current-user` only selects from the app db), but the sandboxed version calls a `get-or-create` style function which seems to create FieldValues that do not yet exist.

I guess I'm confused why existence is a relevant factor for return value. To me, it seems like the existence of FieldValues is a bit of an implementation detail. My thinking is both sandboxed and not should create field values if they don't exist. Can you explain the thinking behind the divergence here?